src/core/lib/security/credentials/ssl/ssl_credentials.h

```diff
@@ -70,6 +70,11 @@ class grpc_ssl_credentials : public grpc_channel_credentials {
                     const grpc_ssl_verify_peer_options* verify_options);
 
   grpc_ssl_config config_;
+  grpc_security_status initialize_client_handshaker_factory(
```

Comment:
micro-nit: For C++, the function name should be capitalized and camel-case, e.g. `InitializeClientHandshakerFactory` rather than `initialize_client_handshaker_factory`. The rest of this file is following C98 style and we should clean it up at some point, but for new code we should definitely follow C++ style.

Reply:
To make sure I'm clear, you want to go on and change this to `InitializeClientHandshakerFactory`? Or just note it and save it for a larger PR where we restyle everything

Reply:
Correct, let's change the naming to `InitializeClientHandshakerFactory`. :)